README.md

```diff
@@ -28,65 +28,72 @@ Alternatively, they can be integrated `annotate-integrate-annotations` as commen
 
 - annotations in org-mode source blocks will be underlined, but the annotations don't show up. This is likely a fundamental incompatibility with the way source blocks are highlighted and the way annotations are displayed.
 
+Portion of the code © 2019 Universita' degli Studi di Palermo
+
```

Comment:
This doesn't work, I'm sorry. If you submit a pull request, you can't have copyright of it.

You can add your changes to the changelog under your name, but there can only be one copyright holder.

Reply:
Oh, i am sorry to hear that :(

I can not remove the copyright, but i am going to fix the fork little by little so you can merge the PR if you change your mind! :)

Anyway thank you for your review, it was very useful to learn how to write an emacs package.

Bye!
C.

Reply:
This source code is published under the terms of the MIT license, which grants every user to the right to

> use, copy, modify, merge, publish, distribute, sublicense, and/or sell
> copies of the Software, and to permit persons to whom the Software is
> furnished to do so, subject to the following conditions:
> 
> The above copyright notice and this permission notice shall be included in
> all copies or substantial portions of the Software.

(See the [LICENSE](https://github.com/bastibe/annotate.el/blob/master/LICENSE) file)

As you can see, you can do as you please with this software, so long as you accept my authorship of it. Thus, the one thing you can not do is to claim copyright of it, which would give you the right to alter its license.

Your contribution will be duly noted in the changelog, and be visible in the Git history. I can even give you commit rights, if you would like to become the maintainer of this software. But even that would not give you the right to claim copyright of it, as you are not the original author.

I will only be able to merge this pull request if you remove that claim. Which is nothing personal, you understand. I approve of your work, and you seem to be a fine human being. But you are not the author of this software, and therefore can not claim copyright for it.